npm/css-src/sass/_card.scss

```diff
@@ -23,3 +23,69 @@
 .tdr-metadata-card .govuk-inset-text {
   color: $govuk-secondary-text-colour;
 }
+
+
+.tdr-tag {
```

Comment:
Is this copied from the GDS?

Reply:
Gareth shared these customized tags for the navigation component.
https://github.com/nationalarchives/tdr-prototype/commit/77b9e5460a88ad0c1e2e46b09f7e6cb5bd5a6f55